internal/hardware/hardware.go

```diff
@@ -108,3 +125,83 @@ func GetMutableHardwareInfoDelta(hardwareMutableInfoPrevious models.HardwareInfo
 
 	return hardwareInfo
 }
+
+// GetDevices recursively traverses a directory specified by path
+// and returns all devices found there.
+// Shamelessly copied from 	https://github.com/opencontainers/runc/blob/main/libcontainer/devices/device_unix.go
+func getDevices(path string) ([]*models.HostDevice, error) {
```

Comment:
Have you considered contributing that code to [assisted installer agent](https://github.com/openshift/assisted-installer-agent)?

Reply:
yes. Initially, I used `Execute("ls"...)` and parse the result but it is not a solid solution. I think this one it is better

Reply:
And why did you decide not to try to add that code to the assisted installer agent?

Reply:
you mean make a PR on installer agent repo?

Reply:
Yes.

Reply:
I didn't think about it